lib/cc/engine/analyzers/engine_config.rb

```diff
@@ -30,6 +30,11 @@ def mass_threshold_for(language)
           end
         end
 
+        def count_threshold_for(language)
+          threshold = fetch_language(language).fetch("count_threshold", nil)
+          threshold ? threshold.to_i : threshold
```

Comment:
This ternary conditional is a bit difficult to read when the switch value is also the default return.

Given the similarity to the `#mass_threshold_for` method above, it might make sense to consolidate:
```ruby
threshold_config_for(:mass, "ruby")
threshold_config_for(:count, "ruby")

def threshold_config_for(option, language)
  threshold = fetch_language(language)["#{option}_threshold"]

  if threshold
   threshold.to_i
  end
end
```